eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -30,14 +30,19 @@ steps:
       Write-Host "This script is not executing on Windows, skipping registry modification."
     }
   displayName: Enable Long Paths if Necessary
-  
-- pwsh: |
-    git clone https://github.com/${{ parameters.TargetDocRepoOwner }}/${{ parameters.TargetDocRepoName }} ${{ parameters.WorkingDirectory }}/repo
-  displayName: Clone Documentation Repository
-  ignoreLASTEXITCODE: false
+
+- template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+  parameters:
+    SkipDefaultCheckout: true
+    Repositories:
+      - Name: ${{ parameters.TargetDocRepoOwner }}/${{ parameters.TargetDocRepoName }}
+        Commitish: master
```

Comment:
We should be careful about starting to hard-code these. It might be worth having some notion of default branch.

Reply:
I could leave it blank, and add some logic into the underlying scripts to detect the default branch when blank, using the code Dozie wrote a while back.

Reply:
@weshaggard updated to determine the default branch.